src/vs/workbench/contrib/scm/browser/dirtydiffDecorator.ts

```diff
@@ -192,8 +192,8 @@ class DirtyDiffWidget extends PeekViewWidget {
 		this._disposables.add(themeService.onDidColorThemeChange(this._applyTheme, this));
 		this._applyTheme(themeService.getColorTheme());
 
-		if (this.model.original) {
-			contextKeyService = contextKeyService.createOverlay([['originalResourceScheme', this.model.original.uri.scheme]]);
+		if (this.model.original && (this.model.original.length > 0)) {
```

Comment:
Since `this.model.original` won't ever be null anymore:

```suggestion
		if (this.model.original.length > 0) {
```